src/btree/bt_delete.c

```diff
@@ -267,7 +269,7 @@ __wt_delete_page_rollback(WT_SESSION_IMPL *session, WT_REF *ref)
      * the update list to be null to be conservative.
      */
     if (current_state == WT_REF_DELETED) {
-        current_state = ref->page_del->previous_ref_state;
```

Comment:
Add a comment here:

When fast truncate succeeds it moves a WT_REF from WT_REF_DISK to WT_REF_DELETED, given that we know that the reverse operation of this is to return the state to WT_REF_DISK.